### DISCUSSION TITLE
Love this project! Just found the extensions! Some idea's...

### DISCUSSION THREAD

**Discussion by jdc4429** [oobabooga/text-generation-webui#1739] [Ideas]
First off, thank you for such a great interface.  I had a few idea's.. since you can just hit enter, could you make it possible to shrink/hide all the buttons on the interface.  I found too much of a gap when you are using all the features like speech to text which is at the bottom of the screen.  Also if it would be possible to get the speech modules to start talking after a sentance completes or maybe a few words if the timing can be corrected to still sound correct.  Right now it takes so long to complete a paragragh that you wait a minute to here the answer as well as see it..  Would be nice if it could be streaming given the time contraints on most hardware.
Maybe you could use the tokens/sec data to estimate when you can start saying the sentance so it doesn't have weird pauses when speaking a sentance.

Also it would be great if we could have the speech recognition on all the time and have a command name to let the AI know when we are talking to it.  Like Hey Siri.. But with whatever name we choose.  That way we don't have to hit record... and it would not matter so much the record option is so far down the screen.

Thanks again for the amazing interface.
Regards,
Jeff

**Comment by GitHub1712** [comment]
+1 with the record button too far below and the continous record waiting for starting command would be usefull.
